extension/custom.css

```diff
@@ -24,6 +24,23 @@ tt,
 .gollum-editor .expanded textarea,
 .gollum-editor .gollum-editor-body,
 .input-monospace,
-.wiki-wrapper .wiki-history .commit-meta code {
+.wiki-wrapper .wiki-history .commit-meta code
+/* +/- pseudo elements on diffs */
+td.blob-code.blob-code-addition:before,
+td.blob-code.blob-code-deletion:before {
 	font-family: 'Ubuntu Mono', 'Droid Sans Mono', 'Operator Mono', 'Liberation Mono', 'Source Code Pro', Menlo, Consolas, Courier, monospace !important;
 }
+
+td.blob-code.blob-code-addition:before,
```

Comment:
These are not highly opinionated changes. Let's put them in `content.css` instead.